Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7.9-slim-buster as base
```

Comment:
You need to update your README with instructions for running the app through Docker.